lib/src/markdown_processor.dart

```diff
@@ -314,15 +314,13 @@ class MarkdownDocument extends md.Document {
     var textContent = _htmlEscape.convert(referenceText);
     var linkedElement = result.commentReferable;
     if (linkedElement != null) {
-      if (linkedElement.href != null) {
+      var href = linkedElement.href;
+      if (href != null) {
```

Comment:
These names are short enough that we could use patterns here.
```dart
if (linkedElement.href case var href?) {}
```
and below
```dart
if (this.href case var href?) {}
```
Maybe even without `this.`?

Reply:
Thanks! Done here